app/src/org/commcare/android/framework/CommCareActivityUIController.java

```diff
@@ -0,0 +1,16 @@
+package org.commcare.android.framework;
+
+/**
+ * Interface to be implemented by any class acting as a ui controller for a CommCareActivity
+ *
+ * IMPORTANT: Any CommCareActivity that uses a CommCareActivityUIController must override
+ * CommCareActivity.initUIController() and CommCareActivity.getUIController()
```

Comment:
Can you update this comment to say WithUIController interface instead of the distinct methods?